### PR TITLE
Style

### PR DESCRIPTION
**Pull request by MariaRiosNavarro**
Add and remove Styles and change some Styles, only styles

[Deployment](https://mobile-simulation-prop-lmnrnrvll-mariariosnavarro.vercel.app/)

### CODE REVIEW

File: components/Form/index.js [hunk 1]
```diff
@@ -78,7 +97,21 @@ export default function Form({ onSubmit, defaultData, formName }) {
     <>
       <form onSubmit={handleSubmit} formName={formName}>
         <StyledHeadingandFoto>
-          {/* NAME INPUT */}
+          <PositioningContainer>
+            <BackLink href={"/contacts"}>{backSVG}</BackLink>
+            {/*-------------------------------- PHOTO INPUT  */}
+            <StyledPhotoLabel htmlFor="photo">
+              Kontact Foto auswählen:
```
Comment (rueveydaakuel) [inline components/Form/index.js]: a small typo, but you can quickly correct it. "Kontakt"

File: components/ContactForm/index.js [hunk 2]
```diff
@@ -76,7 +99,21 @@ export default function Form({ onSubmit, defaultData, formName }) {
     <>
       <form onSubmit={handleSubmit} formName={formName}>
         <StyledHeadingandFoto>
-          {/* NAME INPUT */}
+          <PositioningContainer>
+            <BackLink href={"/contacts"}>{backSVG}</BackLink>
+            {/*-------------------------------- PHOTO INPUT  */}
+            <StyledPhotoLabel htmlFor="photo">
+              Kontact Foto auswählen:
```
Comment (rueveydaakuel) [inline components/ContactForm/index.js]: a small typo, but you can quickly correct it. "Kontakt"

**Review by rueveydaakuel — APPROVED** [review]
Well done Maria 🙌 it looks very good to me; I like it 📱🥳 . I just made a very small comment.

File: components/Form/index.js (hunk 1 above)
Comment (MariaRiosNavarro) [reply components/Form/index.js]: thanks!

File: components/ContactForm/index.js (hunk 2 above)
Comment (MariaRiosNavarro) [reply components/ContactForm/index.js]: Thanks

**Comment by MariaRiosNavarro** [comment]
Please you can check after @rueveydaakuel check. After @rueveydaakuel correction I realised that I had forgotten to make a placeholder in case the user does not provide a photo. That's the main thing in the new code. Also the centering of the Contactform on the contact creation page and on the contact editing page.

[Deployment](https://mobile-simulation-prop-lmnrnrvll-mariariosnavarro.vercel.app/)

File: pages/contacts/[id]/edit/index.js [hunk 3]
```diff
@@ -86,17 +85,15 @@ export default function EditPage() {
...
-          defaultData={contact}
-        />
-        <StyledButtonsContainer>
-          <StyledButton onClick={submitDelete}>loschen</StyledButton>
-        </StyledButtonsContainer>
-        <Navigation />
-      </StyledAppBodyContainer>
+      <ContactForm
+        onSubmit={handleEdit}
+        formName={"edit-contact"}
```
Comment (LukasWeckner) [inline pages/contacts/[id]/edit/index.js]: if "edit-contact" is a string you can get rid of the {}

**Review by LukasWeckner — APPROVED** [review]
Looking good to me :-)

File: pages/contacts/[id]/edit/index.js (hunk 3 above)
Comment (MariaRiosNavarro) [reply pages/contacts/[id]/edit/index.js]: you're right, the habit of putting them everywhere.🤣

**Pull request merged**